src/games/behavmixed.imp

```diff
@@ -643,130 +589,119 @@ T MixedBehaviorProfile<T>::DiffNodeValue(const GameNode &p_node,
 //             MixedBehaviorProfile<T>: Cached profile information
 //========================================================================
 
+
+// compute realization probabilities for nodes and isets.
+template <class T>
+void MixedBehaviorProfile<T>::ComputePass1_realizProbs(const GameNode &node) const
+{
+  if (node->GetParent()) {
+    map_realizProbs[node] = map_realizProbs[(node->GetParent())] * GetActionProb(node->GetPriorAction());
+  }
+  else {
+    map_realizProbs[node] = (T) 1;
+  }
+
+  if (node->GetInfoset()) {
```

Comment:
You don't actually need to check `node->GetInfoset()`; `GetChildren()` would be empty if `GetInfoset()` returns null.